features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/ScopeValidation.jsx

```diff
@@ -106,8 +112,12 @@ export default class ScopeValidation extends React.Component {
      * @memberof ScopeValidation
      */
     componentDidMount() {
-        const { resourcePath: currentResourcePath, resourceMethod: currentResourceMethod } = this.props;
-        const hasScope = ScopeValidation.hasScopes(currentResourcePath, currentResourceMethod);
+        const {
+            resourcePath: currentResourcePath,
+            resourceMethod: currentResourceMethod,
+            client: currentClient,
```

Comment:
We can make API_CLIENT the default client so the existing <ScopeValidation/> usages won't change, and override the prop for the service catalog client, This way we can reduce the changes and make `ScopeValidation` simple for API client

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/10387/commits/c61d1b858af384f477b29355f7a70f7675ae9090